bin/build-docker-image

```diff
@@ -1,5 +1,10 @@
 #!/usr/bin/env bash
 
+if [ $# -eq 0 ] ; then
+  echo "Usage: $0 dockerhubusername"
```

Comment:
This should say something about what this script does.  E.g., does it build the entire docker image and upload it to docker hub under `dockerhubusername/pageshot` ?